docs/emr/configuration-and-execution.rst

```diff
@@ -0,0 +1,16 @@
+===========================
+Configuration and Execution
+===========================
+
+- Configuring Presto Admin on EMR cluster should proceed in the same manner as on any other cluster (see :ref:`presto-admin-configuration-label`) except for the following caveat:
+
+	- We recommend designating the EMR master node as the Presto coordinator during Presto Admin configuration.
+
+- Running Presto Admin on EMR cluster should proceed in the same manner as on any other cluster (see sections starting from :ref:`presto-server-installation-label`  onwards in :ref:`quick-start-guide-label`) except for the following caveats:
+
+	- The default version of Java installed on EMR cluster (up to EMR 4.4.0) is 1.7, whereas Presto requires Java 1.8. Install Java 1.8 on EMR cluster by following the instructions in Java installation (see :ref:`java-installation-label`).
+
+	- This is relevant only if you have used an Amazon EC2 key pair while provisioning the Amazon EMR cluster. Setting up passwordless SSH for the default ``hadoop`` user on an EMR instance is not effective for running Presto Admin commands. Instead, you need to do the following: 
+                * Copy the ``.pem`` file associated with the key pair to the Presto Admin installation node of the EMR cluster.
+                * Use ``-i <path to .pem file>`` input argument when running presto-admin commands on the node.
+
```

Comment:
we should probably link to the ssh passwordless pa doc where we explain usage of -i argument?